src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs

```diff
@@ -18,6 +18,11 @@ public LoggerPhaseScope(string phaseName)
         }
 
         public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
+            : this(phaseName, enablePerformanceScope, LogLevel.Diagnostic)
+        {
+        }
+
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope, LogLevel perfLogLevel)
```

Comment:
@vwxyzh updated, please take a look.

---
In reply to: [94709960](https://github.com/dotnet/docfx/pull/1117#discussion_r94709960) [](ancestors = 94709960)